config/locales/stamp_duty.en.yml

```diff
@@ -36,7 +36,7 @@ en:
         link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-transaction-tax-calculator-wales"
       scotland:
         title: Calculate Land and Buildings Transaction Tax for Scotland
-        link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-and-and-buildings-transaction-tax-calculator-scotland"
+        link: "https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-and-buildings-transaction-tax-calculator-scotland"
```

Comment:
This looks good. 

Although not explicitly stated in the TP card I think there will need to be a similar change made it he corresponding `.cy` file. Might be worth checking but it is probably also wrong.

Reply:
Thanks